project/JsonMaker.java

```diff
@@ -0,0 +1,35 @@
+/**
+* JsonMaker reads from several text files that show information about selected
+* words in popular topics where misinformation is prevelant. List of words,
+* word facts, images, and source URLs for the words chosen are organized in
+* their respective files. JsonMaker transforms this data into a easy-to-read
+* and manage JSON file, for example:
+* {
+*   "covid": {
+*     "description": "People of any age, even healthy young adults and children, can get COVID-19. Risk for severe illness with COVID-19 increases with age, with older adults at highest risk. Certain medical conditions can also increase risk for severe illness.",
+*     "image": "/images/covid.png",
+*     "URL": [
+*       "https://www.cdc.gov/coronavirus/2019-ncov/your-health/need-to-know.html",
+*       "https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/older-adults.html"
+*     ]
+*   },
+*   "climate change": {
+*     "description": "People who study Earth see that Earth's climate is getting warmer. Earth's temperature has gone up about one degree Fahrenheit in the last 100 years. This may not seem like much. But small changes in Earth's temperature can have big effects.",
+*     "image": "/images/climate-change.png",
+*     "URL": [
+*       "https://www.nasa.gov/audience/forstudents/k-4/stories/nasa-knows/what-is-climate-change-k4.html",
+*       "https://climate.nasa.gov/news/3072/direct-observations-confirm-that-humans-are-throwing-earths-energy-budget-off-balance/"
+*     ]
+*   },
+*   ... more words ...
+* }
+*
+*
+* @author : Samuel Rodriguez (samuelrprofessional@gmail.com)
+* Date : 03/30/2021
+*/
+// TODO: imports
```

Comment:
It is usually useful to use the following:

TODO (username, in parenthesis like so): what there is todo and when it is expected to be resolved.

Reply:
That is a better method, I will use that format when I put more TODOs!